airbyte-local.sh

```diff
@@ -331,8 +332,29 @@ function writeDstConfig() {
 function getConfigFromWizard() {
     local docker_image=$1
     local config_filename=$2
-    docker run -it --rm -v "$tempdir:/configs" "$docker_image" airbyte-local-cli-wizard \
-      --json "/configs/$config_filename"
+    if [[ $docker_image == farosai/* ]]; then
+        docker run -it --rm -v "$tempdir:/configs" "$docker_image" airbyte-local-cli-wizard \
+        --json "/configs/$config_filename"
+    else
+        docker run -it --rm "$docker_image" spec \
+        > "$tempdir/$spec_filename"
+
+        if [[ $src_docker_image == farosai/* ]]; then
+            wizard_docker_image="$src_docker_image"
+            log "Using source image $wizard_docker_image to run wizard"
+        elif [[ $dst_docker_image == farosai/* ]]; then
+            wizard_docker_image="$dst_docker_image"
+            log "Using destination image $wizard_docker_image to run wizard"
+        else
+            # Use an arbitrary farosai image to run the wizard
```

Comment:
how is it "arbitrary" if you are forcing the use of `farosai/airbyte-faros-graphql-source` here?

Reply:
By arbitrary I meant any `farosai/*` connector would do, so I chose one at random.

This only happens when neither the src/dst are Faros AI connectors